core/src/main/java/org/elasticsearch/action/admin/indices/analyze/TransportAnalyzeAction.java

```diff
@@ -156,8 +155,6 @@ protected AnalyzeResponse shardOperation(AnalyzeRequest request, ShardId shardId
                  **/
                 if (indexService != null) {
                     field = indexService.getIndexSettings().getDefaultFields().get(0);
-                } else {
-                    field = AllFieldMapper.NAME;
```

Comment:
If indexService is null, the request will fallback to the `standard` analyzer so it should be ok.